query-node/mappings/src/content/app.ts

```diff
@@ -49,32 +52,19 @@ export async function processCreateAppMessage(
     platforms: appMetadata?.platforms || undefined,
     category: appMetadata?.category || undefined,
     authKey: appMetadata?.authKey || undefined,
-    ownerCuratorGroup: channel.ownerCuratorGroup,
-    ownerMember: channel.ownerMember,
-    channel,
   })
   await store.save<App>(newApp)
   logger.info('App has been created', { name })
 }
 
-async function createAppId(event: SubstrateEvent): Promise<string> {
-  return `${event.blockNumber}-${event.indexInBlock}`
-}
-
-export async function processUpdateApp(
-  store: DatabaseManager,
-  channelId: ChannelId,
-  message: IUpdateApp
-): Promise<void> {
+export async function processUpdateAppMessage(store: DatabaseManager, message: IUpdateApp): Promise<void> {
   const { appId, appMetadata } = message
 
-  const app = await getAppById(store, appId)
+  const app = await getAppByIdAndMemberId(store, appId)
 
   if (!app) {
-    inconsistentState("App doesn't exists; appId:", appId)
-  }
-  if (app?.channel.id !== channelId.toString()) {
-    inconsistentState(`Cannot update app; app does not belong to the channelId: `, channelId)
+    logger.error("App doesn't exists or doesn't belong to the member", { appId })
+    return
```

Comment:
Same issue as previously pointed out: returning nothing is equivalent to success condition